config/webpack/environment.js

```diff
@@ -5,8 +5,10 @@ const webpack = require("webpack");
 environment.plugins.append(
   "Provide",
   new webpack.ProvidePlugin({
-    $: "jquery",
-    jQuery: "jquery",
+    $: 'jquery/src/jquery',
```

Comment:
I think all we need is this line. Test it out.

Reply:
Yeah!! is still working!!
Done!